README.md

```diff
@@ -4,7 +4,6 @@ SAI testing and integration framework that is based on SONiC sairedis project.
 ```sh
 git clone https://github.com/PLVision/sai-challenger.git
 cd sai-challenger/
-git submodule update --init --recursive
```

Comment:
pip allows to install dependencies using git, as it is a python package with setup file, moved it to pip (requirements.txt)